lib/paystackex/sdk/client.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Paystackex.Sdk.Client do
+  use Ext.Sdk.BaseClient, endpoints: Map.keys(Paystackex.Sdk.Config.data().endpoints)
```

Comment:
We just implemented unified BaseClient which resolves all our requirements during last year.